migrations/20240426200342_new_movies_created_are_set_to_draft_status_by_default/migration.sql

```diff
@@ -0,0 +1,2 @@
+-- AlterTable
+ALTER TABLE `movie` MODIFY `status` VARCHAR(191) NULL DEFAULT 'draft';
```

Comment:
When I try to run `npm run dev` on this branch it warns me it's about to delete a table `movie`, whereas our table name is `Movie` - note the capital M. Let's fix this to make sure prisma knows we're talking about the right table.

Reply:
@tomershvueli Thank you!! I'm definitely learning here :) I have updated and pushed with changes.